### PR TITLE
Dashboard skeleton

### PR DESCRIPTION
**Pull request by KaminKevCrew**
Here's the dashboard skeleton. Basic components are integrated and buttons function in navbar. 

Still todo: 
Styling the navbar so it looks correct. It's currently a little bit bunched up.
Setting up profile pic/username functionality once backend is fully set up.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -15,8 +20,13 @@ function App() {
       <BrowserRouter>
         <Switch>
           <Route path="/" exact component={Signup} />
-          <Route path="/gmail" component={SimpleDialogDemo} />
+          <Route oath="/" component={LoggedInNavBar} />
```
Comment (CharlieLe3) [inline client/src/App.js]: Think a typo happened here

**Review by CharlieLe3 — COMMENTED** [review]
Just a typo issue

File: client/src/App.js (hunk 1 above)
Comment (KaminKevCrew) [reply client/src/App.js]: You are correct, thanks for the catch!

**Review by kimcarey — DISMISSED** [review]
Looks good to me! I'll plan to build on top of it

**Comment by kimcarey** [comment]
I loaded your branch locally and got this error (warning).
```
Proxy error: Could not proxy request /demo_avatar.png from localhost:3000 to http://localhost:5000.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNREFUSED).
```
Looks like your may have forgotten to add the user icon? I don't see that file anywhere in the files, but it's referenced here `<Avatar alt="John Doe" src="demo_avatar.png" />`

**Review by kimcarey — CHANGES_REQUESTED** [review]
Seeing the updated navbar and the links change the url (good work!) but the placeholder content of the pages doesn't seem to work as they are just blank except the login page?
Screen Shot 2020-07-28 at 10 32 54 PM

**Review by kimcarey — APPROVED** [review]
LGTM! I just switched to your branch and re-ran the server and I'm seeing the div messages on each of the pages. Whoo hoo!!

**Review by CharlieLe3 — APPROVED** [review]

**Pull request merged**